userspace/libscap/engine/modern_bpf/scap_modern_bpf.c

```diff
@@ -62,6 +67,126 @@ static int32_t populate_64bit_interesting_syscalls_table(bool* ppm_sc_array)
 	return ret;
 }
 
+static bool read_next_kernel_config_option(gzFile file, char *buf, size_t n,
+					   char **value)
+{
+	char *sep;
+
+	while (gzgets(file, buf, n)) {
+		if (strncmp(buf, "CONFIG_", 7))
+			continue;
+
+		sep = strchr(buf, '=');
+		if (!sep)
+			continue;
+
+		/* Trim ending '\n' */
+		buf[strlen(buf) - 1] = '\0';
+
+		/* Split on '=' and ensure that a value is present. */
+		*sep = '\0';
+		if (!sep[1])
```

Comment:
[nitpick] Since we are testing for a char, I find it clearer expressed this way, but both are fine
```suggestion
		if (sep[1] == '\0')
```